tokio-util/src/lib.rs

```diff
@@ -115,6 +115,12 @@ mod util {
 
         let n = {
             let dst = buf.chunk_mut();
+
+            // FIXME(https://github.com/tokio-rs/bytes/pull/548): Consider switching to the safe
+            // `chunk_mut().into::<&mut [MaybeUninit<u8>]>()` if this PR lands.
+            //
+            // Safety: `chunk_mut()` returns a `&mut UninitSlice`, and `UninitSlice` is a
```

Comment:
this is kind of a minor side note, but i've noticed that Tokio contains a number of "safety" comments, but the capitalization of the word "safety" is not very consistent. i think we may want to make this more consistent so that it's easier to grep for all such "safety" comments in the repo.

out of interest, i did a quick search to see which capitalization was the most common:
```shell
:# eliza at noctis in tokio on  master [$] via ⚙️ v1.60.0 in ❄️ nix-shell [impure]
:; for needle in "safety:" "SAFETY:" "Safety:"; do echo -n "$needle " ; rg "$needle" | wc -l ; done
safety: 51
SAFETY: 42
Safety: 126
```
and it looks like "Safety:" is the clear winner, so this seems consistent with the rest of the code.

we should really consider changing the other "safety" comments to be consistent with this...